de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/callbacks/TagsSearchCallback.java

```diff
@@ -0,0 +1,14 @@
+package org.iplantc.de.diskResource.client.presenters.callbacks;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+import jsinterop.annotations.JsFunction;
+
+/**
+ * Created by sriram on 3/9/18.
+ */
+@JsFunction
+@SuppressWarnings("unusable-by-js")
+public interface TagsSearchCallback {
+    void searchComplete(Splittable[] tags);
```

Comment:
I'm probably missing something, but is there a reason for `Splittable[]` over `Splittable`?

Reply:
I used `Splittable` array since there will be more than one tag returned. Each `Splittable` represents a`tag`. The Datasource for the MUI component I used requires an array.